tests/testthat/test-model_parameters.mixed.R

```diff
@@ -6,11 +6,14 @@ test_that("model_parameters.mixed", {
   params <- model_parameters(insight::download_model("lmerMod_1"), standardize = "refit")
   testthat::expect_equal(c(nrow(params), ncol(params)), c(2, 8))
 
+  params <- model_parameters(insight::download_model("lmerMod_1"), standardize = "full", ci = c(0.8, 0.9))
+  testthat::expect_equal(c(nrow(params), ncol(params)), c(2, 10))
+
   params <- model_parameters(insight::download_model("merMod_1"), standardize = "refit")
   testthat::expect_equal(c(nrow(params), ncol(params)), c(2, 8))
 
   params <- model_parameters(insight::download_model("merMod_2"), standardize = "refit")
   testthat::expect_equal(c(nrow(params), ncol(params)), c(3, 8))
 
-  # TODO: Not sure how to deal with bootstrapped mixed models... As it throws a reasonable amount of singular fits...
+  # TODO: Not sure how to deal with bootstrapped mixed models... As it throws an unreasonable amount of singular fits...
 })
```

Comment:
The mtcars data set is probably no good choice for testing in general, because it has no real "binary" variables to test logistic regression, nor enough cases for mixed models. If you bootstrap, you even get duplicates from these few cases, which causes all the singular fits.

We should think of also using other data sets...